shotgun_api3/shotgun.py

```diff
@@ -850,7 +850,10 @@ def find_one(self, entity_type, filters, fields=None, order=None, filter_operato
             Defaults to ``["id"]``.
         :param int order: Optional list of fields to order the results by. List has the format::
 
-            [{'field_name':'foo', 'direction':'asc'}, {'field_name':'bar', 'direction':'desc'}]
+                [
+                 {'field_name':'foo', 'direction':'asc'},
+                 {'field_name':'bar', 'direction':'desc'}
+                ]
```

Comment:
Maybe the indentation can be improved:
```suggestion
                [
                    {'field_name':'foo', 'direction':'asc'},
                    {'field_name':'bar', 'direction':'desc'},
                ]
```

Reply:
Just push that suggestion...
![Screenshot 2023-03-13 at 4 46 13 PM](https://user-images.githubusercontent.com/119680820/224828829-fabfb139-c432-4cad-ad04-d3e1fc0d2314.png)